Raven.Database/Config/InMemoryRavenConfiguration.cs

```diff
@@ -1588,6 +1590,10 @@ public class ReplicationConfiguration
             /// Indicates how many seconds replication task will wait before propagating replication documents
             /// </summary>
             public int ReplicationPropagationDelayInSeconds { get; set; }
+
+            public string ReplicationToV4CertificatePath { get; set; }
```

Comment:
Can we find a better name? What if we will have V5?

Reply:
done